src/Microsoft.IdentityModel.Protocol.Extensions/Properties/AssemblyInfo.cs

```diff
@@ -34,4 +34,4 @@
 [assembly: NeutralResourcesLanguage("en-US")]
 [assembly: InternalsVisibleTo("Microsoft.IdentityModel.Protocol.Extensions.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100B5FC90E7027F67871E773A8FDE8938C81DD402BA65B9201D60593E96C492651E889CC13F1415EBB53FAC1131AE0BD333C5EE6021672D9718EA31A8AEBD0DA0072F25D87DBA6FC90FFD598ED4DA35E44C398C454307E8E33B8426143DAEC9F596836F97C8F74750E5975C64E2189F45DEF46B2A2B1247ADC3652BF5C308055DA9")]
 [assembly: AssemblyMetadata("Serviceable", "True")]
-[assembly: AssemblyFileVersion("1.0.REPLACE_FILE_VERSION_AT_BUILD_TIME")]
```

Comment:
Scripts will add the this similar to 
"[assembly: AssemblyInformationalVersionAttribute(""$hash"")]"